### ISSUE TITLE
Switch language for counter print (\arabic and \alph)

### ISSUE THREAD

**Issue by dolfim** [reutenauer/polyglossia#133]
I have the following environment:

``` latex
\newenvironment{QTF}[1]
{\begin{framed}
  {\noindent\bf \textenglish{ \Alph{ptnum}\arabic{qtnum2} (#1 pt) }}}
{
\end{framed}
  \addtocounter{qtnum2}{1}}
```

I would like this to print `ptnum` and `qtnum2` always in english:
- A1 (2 pt)
- A2 (1 pt)
- B1 (1 pt)
- ...

When the main language is something like arabic, which comes with its own numbering, the output of `\arabic` and `\Alph` is not converted, even though it is inside a `\textenglish` block.

**Comment by dolfim** [comment]
Is this related to #119?

**Comment by jspitz** [comment]
This should be fixed with #238.

**Comment by jspitz** [comment]
Someone who can please close this ticket.
